stdlib/2and3/imaplib.pyi

```diff
@@ -19,8 +25,8 @@ class IMAP4:
     debug: int = ...
     state: str = ...
     literal: Optional[Text] = ...
-    tagged_commands: Dict[str, str] = ...
-    untagged_responses: Dict[str, str] = ...
+    tagged_commands: Dict[str, Optional[bytes]]
```

Comment:
I think the keys for this are bytes as well, https://github.com/python/cpython/blob/master/Lib/imaplib.py#L1195

Reply:
Good catch, thanks!